src/Core/NounCategories.fs

```diff
@@ -0,0 +1,14 @@
+﻿module NounCategories
```

Comment:
Is it just for nouns? Afaik the same categories apply to adjectives, right?

Reply:
Fair point, yes, e.g. "grammar categories" is better and flexible.